tests/conftest.py

```diff
@@ -22,6 +22,24 @@
 
 __all__ = 'IsNow', 'TestEnv', 'ClientWithHandler', 'try_import'
 
+import dirty_equals
+
+OldIsNow = dirty_equals.IsNow
+
+
+def newIsNow(*a: Any, **ka: Any):
+    # this has a big effect on the test runtime
+    # tests are passing in both cases
+
+    # ~ 4min
+    # (the delta is required to make the tests work)
+    # return OldIsNow(*a,**ka,delta=50)
+
+    # 16 sec
+    return dirty_equals.AnyThing()
```

Comment:
@15r10nk what if we got rid of the use of `IsNow()` and just did `timestamp=dirty_equals.AnyThing()`? Would that help?

Reply:
I would not remove all `IsNow()`s from the tests yet. I think there is some bug inside inline-snapshot which causes this and I want to try to fix it first.

But you can use the workaround above for now if you want.